lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -234,6 +236,10 @@ Object.defineProperties(
         routes,
         id: userConfig.id,
         metrics: userConfig.metrics || false,
+        throttle: {
+          burstLimit: (userConfig.throttle && userConfig.throttle.burstLimit) || -1,
```

Comment:
Unfortunately we need to be more verbose, as I believe we should also support `0` (?)

in that case we should probably upfront resolve values into vars and then assign as:

```javascript
const throttleBurstLimit = _.get(userConfig.throttle, 'burstLimit');
...
thorttle: {
  burstLimit: throttleBurstLimit == null ? -1 : throttleBurstLimit
  ...
}
```